components/service/glean/src/main/java/mozilla/components/service/glean/EventMetricType.kt

```diff
@@ -10,7 +10,7 @@ import mozilla.components.support.base.log.logger.Logger
 // Maximum length of any string value in the extra dictionary, in UTF8 byte sequence length.
 internal const val MAX_LENGTH_EXTRA_KEY_VALUE = 80
 // Maximum length of any passed value string, in UTF8 byte sequence length.
-internal const val MAX_LENGTH_VALUE = 80
+internal const val MAX_LENGTH_EVENT_VALUE = 80
```

Comment:
Are we renaming due to name clashing from the consts in `StringMetricType`? If so, maybe a better option is to use [companion objects](https://kotlinlang.org/docs/reference/object-declarations.html#companion-objects) to hold the constants (see [here](https://github.com/mozilla-mobile/android-components/blob/c874c343cab18fe64f45ba8eb9a726f829fe06b9/components/service/fretboard/src/main/java/mozilla/components/service/fretboard/DeviceUuidFactory.kt#L40-L43) for an example)? I'm not a language expert, so I'm not sure if there are implications in doing this. Maybe @pocmo knows better! Regardless, I'd stick with your renaming in this PR and eventually change this in a new bug/PR, so we're not blocked.

Reply:
Yep -- I was surprised that these constants were in the same namespace (and thus clashed).  I think members of the class is probably better, but I can do that as a follow-on to this.